packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -111,6 +121,12 @@ const Dialog = ({
       children
     );
 
+  const actionButtons = [
+    tertiaryCallToActionComponent || callToActionLastComponent,
+    secondaryCallToActionComponent || callToActionSecondComponent,
+    primaryCallToActionComponent || callToActionFirstComponent,
+  ].filter(Boolean);
```

Comment:
I was gonna say this needs to be memoized, but looking at it again, it actually **doesn't**! 👍

Reply:
Can you write a comment on why we have `||`? It's not clear, and I'd "clean that up" by removing the `||`s. Sadly, we really need those `||`, or it doesn't work correctly.